azblob/url_blob.go

```diff
@@ -267,7 +308,7 @@ func (b BlobURL) StartCopyFromURL(ctx context.Context, source url.URL, metadata
 		nil, // Blob tags
```

Comment:
should this comment remain? I can see many other places like that plz check them.